doc/CODE_OF_CONDUCT.md

```diff
@@ -1,11 +1,12 @@
-# Code of Conduct - EcoCode
+# Code of Conduct - Creedengo
 
-- [Our Pledge](#our-pledge)
-- [Our Standards](#our-standards)
-- [Our Responsibilities](#our-responsibilities)
-- [Scope](#scope)
-- [Enforcement](#enforcement)
-- [Attribution](#attribution)
+- [Code of Conduct - Creedengo](#code-of-conduct---creedengo)
```

Comment:
sorry ... but the title is considered in table ... maybe change the style of title to only have real documentation section and not the title in table of contents